modules/kernel/src/main/java/org/opencastproject/kernel/security/SystemTokenRememberMeUtils.java

```diff
@@ -44,10 +44,19 @@ public final class SystemTokenRememberMeUtils {
 
   private static final Logger logger = LoggerFactory.getLogger(SystemTokenRememberMeUtils.class);
 
+  /** This is the default cookie key, that is configured in Opencast **/
+  private static final String defaultCookieKey = "opencast";
```

Comment:
As a class-wide constant I suggest using an uppercase name as usual:

```suggestion
  private static final String DEFAULT_COOKIE_KEY = "opencast";
```

Reply:
Done